scripts/aks-auto-start.sh

```diff
@@ -41,3 +50,12 @@ jq -c '.[]' <<< $SUBSCRIPTIONS | while read subscription; do
 
     done
 done
+
+jq -c '.[]' <<< $SUBSCRIPTIONS | while read subscription; do
+subscription
+    jq -c '.[]' <<< $CLUSTERS | while read cluster; do
+        cluster
+        echo $NAME
+        az aks show --name  $NAME -g $RESOURCE_GROUP | jq -r .powerState.code
```

Comment:
could you try this please?
```suggestion
        ts_echo $NAME
        RESULT=$(az aks show --name  $NAME -g $RESOURCE_GROUP | jq -r .powerState.code)
        ts_echo "${RESULT}"
```

Reply:
the output from above:

13:57:30 About to start cluster cft-ptlsbox-00-aks (rg:cft-ptlsbox-00-rg) 
13:57:30 Test that plum works in sandbox after cft-ptlsbox-00-aks start-up 
13:57:30 plum does not work in sandbox after cft-ptlsbox-00-aks start-up 
13:57:41 About to start cluster cft-sbox-00-aks (rg:cft-sbox-00-rg) 
13:57:41 Test that plum works in Sandbox after cft-sbox-00-aks start-up 
13:57:41 plum does not work in Sandbox after cft-sbox-00-aks start-up 
13:57:41 About to start cluster cft-sbox-01-aks (rg:cft-sbox-01-rg) 
13:57:41 Test that plum works in Sandbox after cft-sbox-01-aks start-up 
13:57:41 plum does not work in Sandbox after cft-sbox-01-aks start-up 
13:57:45 About to start cluster ss-sbox-00-aks (rg:ss-sbox-00-rg) 
13:57:45 Test that toffee works in sandbox after ss-sbox-00-aks start-up 
13:57:45 toffee does not work in sandbox after ss-sbox-00-aks start-up 
13:57:45 About to start cluster ss-sbox-01-aks (rg:ss-sbox-01-rg) 
13:57:45 Test that toffee works in sandbox after ss-sbox-01-aks start-up 
13:57:45 toffee does not work in sandbox after ss-sbox-01-aks start-up 
13:57:46 About to start cluster cft-ithc-00-aks (rg:cft-ithc-00-rg) 
13:57:46 Test that plum works in ithc after cft-ithc-00-aks start-up 
13:57:46 plum does not work in ithc after cft-ithc-00-aks start-up 
13:57:46 About to start cluster cft-ithc-01-aks (rg:cft-ithc-01-rg) 
13:57:46 Test that plum works in ithc after cft-ithc-01-aks start-up 
13:57:46 plum does not work in ithc after cft-ithc-01-aks start-up 
13:57:52 About to start cluster ss-test-00-aks (rg:ss-test-00-rg) 
13:57:52 Test that toffee works in testing after ss-test-00-aks start-up 
13:57:52 toffee does not work in testing after ss-test-00-aks start-up 
13:57:52 About to start cluster ss-test-01-aks (rg:ss-test-01-rg) 
13:57:52 Test that toffee works in testing after ss-test-01-aks start-up 
13:57:52 toffee does not work in testing after ss-test-01-aks start-up 
13:57:53 About to start cluster cft-demo-01-aks (rg:cft-demo-01-rg) 
13:57:53 Test that plum works in demo after cft-demo-01-aks start-up 
13:57:53 plum does not work in demo after cft-demo-01-aks start-up 
13:59:02 cft-ptlsbox-00-aks 
13:59:03 Running 
13:59:14 cft-sbox-00-aks 
13:59:15 Running 
13:59:15 cft-sbox-01-aks 
13:59:16 Running 
13:59:20 ss-sbox-00-aks 
13:59:21 Running 
13:59:21 ss-sbox-01-aks 
13:59:22 Running 
13:59:24 cft-ithc-00-aks 
13:59:25 Running 
13:59:25 cft-ithc-01-aks 
13:59:27 Running 
13:59:32 ss-test-00-aks 
13:59:33 Running 
13:59:33 ss-test-01-aks 
13:59:34 Running 
13:59:35 cft-demo-01-aks 
13:59:37 Running